modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version 5.4.0 (TBD)
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
+    Fixed filters for screening commands from the undo stack to be properly case-insensitive.
```

Comment:
Sorry to be a pain @trevorjay, but I just noticed the JIRA number is missing from the end of this, so it should probably be:
```
 Fixed filters for screening commands from the undo stack to be properly case-insensitive #TINY-5946
```

Reply:
No worries. That's exactly why I was happy to be able to do a "dry run" with an almost non-patch like this. Thanks!